src/components/FeaturedBlogPost.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import Date from './Date'
+import FeaturedImage from './FeaturedImage'
+import Tag from './Tag'
+import Title from './Title'
+import TruncatedParagraph from './TruncatedParagraph'
+
+const BlogPostWrapper = styled.section`
+  margin: 20px;
+  width: 477px;
+  align-content: flex-end;
+`
+
+
+const BlogPost = ({ post }) => (
+  <BlogPostWrapper>
+    <Date date={post.publishedDate}/>
+    {post.tags.map((tag) => (
+      <Tag tag={tag} />
```

Comment:
same: `<Tag />` might need a a key prop https://reactjs.org/docs/lists-and-keys.html